src/v1/products.ts

```diff
@@ -345,6 +354,29 @@ export class Products extends ThBaseHandler {
     }
   }
 
+  async bulkCreate (products: Product[], query?: HandlerProductsQuery): Promise<ProductsBulkResponse> {
+    const base = this.uriHelper.generateBaseUri('/bulk_create')
+    const uri = this.uriHelper.generateUriWithQuery(base, query)
+
+    try {
+      const response = await this.http.getClient().post(uri, products)
+      if (![200, 422].includes(response.status)) {
```

Comment:
Why are you treating `422` not as an error but the same as `200`?

Reply:
Because we give back 422 in case they didnt send products, or were all invalid, so nothing was created. From the point of all were invalid, maybe would make sense another error.

Reply:
Definitely a different error. 422 is always used for a bad request that results from unknown fields.
409 might be a better one that is frequently used.